gnoi_cert/gnoi_cert.go

```diff
@@ -122,17 +127,36 @@ func gnoiEncrypted(c tls.Certificate) (*grpc.ClientConn, *cert.Client) {
 
 // gnoiAuthenticated creates an authenticated TLS connection to the target.
 func gnoiAuthenticated(targetName string) (*grpc.ClientConn, *cert.Client) {
-	clientEnt, err := entity.CreateSigned("client", nil, caEnt)
-	if err != nil {
-		log.Exitf("Failed to create a signed entity: %v", err)
+	var signedCert tls.Certificate
```

Comment:
This logic already existds here? I think you can use it unless I am missing something:
https://github.com/google/gnxi/blob/master/utils/credentials/credentials.go#L103

Reply:
I could but use it but I'd have to add some return paths that won't be used by other clients (for caEnt)

Reply:
yep, as suggested above! =)